src/DebugEngineHost.Stub/project.json

```diff
@@ -4,6 +4,7 @@
     "dnxcore50.app": {}
   },
   "dependencies": {
+    "MicroBuild.Core": "0.2.0",
```

Comment:
My guess: This adds MicroBuild.Core dependency to three projects that probably don't actually need the dependency at all, and doesn't add it to the project that actually needs the dependency (MIDebugPackage)